src/sketch.js

```diff
@@ -212,8 +223,31 @@ function draw()
     // set background color
     background(...BACKGROUND_COLOR);
 
+    const step_params = {x: 0.2, y: 0.2, speed: 0.18}
+
+    if(soundOn)
+    {
+        // vary terrain speed based on sound amplitude
+        step_params.speed = map(amplitude.getLevel(), 0, 1, AUDIO_TO_SPEED.min, AUDIO_TO_SPEED.max);
+
+        // change the sound pan
+        alterPan();
+
+        let a = fft.analyze();
+
+        BACKGROUND_COLOR[0] = (a[0] + a[1]) / 2;
```

Comment:
Please comment this block, also, is there a better name to this than `a`?

Reply:
👍